tools/go.mod

```diff
@@ -15,3 +15,5 @@ replace (
 	github.com/deckhouse/deckhouse/dhctl => ../dhctl
 	github.com/deckhouse/deckhouse/go_lib/cloud-data => ../go_lib/cloud-data
 )
+
+replace github.com/flant/shell-operator => github.com/flant/shell-operator v1.3.2-0.20230911172604-d965e9a98cd7
```

Comment:
I'm confused with this one. Why do we need this replace?

Reply:
It is not necessary. Leftovers from experimenting with versions shell-operators.
I'll be more careful in the future.